org.eclipse.lemminx/src/main/java/org/eclipse/lemminx/services/extensions/IHoverParticipant.java

```diff
@@ -41,5 +41,13 @@
 	 * @return the Value of MarkupContent {@link String}
 	 */
 	String onAttributeValue(IHoverRequest request) throws Exception;
+	
+	/**
+	 * onTagText method
+	 *
+	 * @param hoverRequest the hover request.
+	 * @return the Value of MarkupContent {@link String}
+	 */
+	String onTagText(IHoverRequest resquest) throws Exception;

```

Comment:
Should this maybe be renamed to "onText"? What do you think @angelozerr

Reply:
Yes onText is better (it's a w3c DOM Text)

Reply:
Ok will update it to onText

Reply:
Please implement this method in HoverParticipantAdapter

Reply:
Thanks